ml/rlvr/fine_tuning_orchestrator.py

```
@@ -0,0 +1,463 @@
+
+        Returns:
+            Fine-tuning run details
+        """
+        # Check if enough training data
+        stats = await self.collector.get_collection_stats(agent_id)
+        if not stats['ready_for_training']:
+            raise ValueError(
+                f"Not enough training data. "
+                f"Need {self.collector.min_examples_per_agent}, "
+                f"have {stats['total_examples']}"
+            )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)